packages/client/src/contracts/StreamRegistry.ts

```diff
@@ -71,7 +71,7 @@ const CACHE_KEY_SEPARATOR = '|'
 export class StreamRegistry {
 
     private streamRegistryContract?: ObservableContract<StreamRegistryContract>
-    private streamRegistryContractsReadonly: ObservableContract<StreamRegistryContract>[]
+    private streamRegistryContractsReadonly: ObservableContract<StreamRegistryContract>
```

Comment:
Rename (to singular)
```suggestion
    private streamRegistryContractReadonly: ObservableContract<StreamRegistryContract>
```